chapter-06/src/main/scala/chapter06/chat_schema.scala

```diff
@@ -1,24 +1,26 @@
 package chapter06
 
+import slick.driver.JdbcDriver
```

Comment:
The upgrade guide indicates we should no longer use the JdbcDriver - http://slick.typesafe.com/doc/3.0.0/upgrade.html#jdbcdriver